Source/WebCore/rendering/updating/RenderTreeUpdaterGeneratedContent.cpp

```diff
@@ -318,9 +318,15 @@ void RenderTreeUpdater::GeneratedContent::updateWritingSuggestionsRenderer(Rende
 
     auto textWithoutSuggestion = nodeBeforeWritingSuggestionsTextRenderer->text();
 
+#if PLATFORM(IOS_FAMILY)
+    auto prefix = textWithoutSuggestion;
+    auto suffix = ""_s;
+#else
     auto offset = writingSuggestionData->offset();
+
     auto prefix = textWithoutSuggestion.substring(0, offset);
     auto suffix = textWithoutSuggestion.substring(offset);
+#endif
```

Comment:
I think it would be good to refactor this code, so that we avoid this platform-specific `#define` in core rendering code in WebCore.

Maybe we could source it from the already-platform-specific venues in `Editor{IOS|Mac}.mm` and then propagate it here, or add a new `EditingBehavior` method.